helper/src/commonMain/kotlin/com/algolia/instantsearch/helper/filter/range/FilterRangeConnectionSearcher.kt

```diff
@@ -0,0 +1,73 @@
+@file:Suppress("FunctionName")
+
+package com.algolia.instantsearch.helper.filter.range
+
+import com.algolia.instantsearch.core.connection.Connection
+import com.algolia.instantsearch.core.connection.ConnectionImpl
+import com.algolia.instantsearch.core.number.range.Range
+import com.algolia.instantsearch.helper.filter.range.internal.mapper
+import com.algolia.instantsearch.helper.searcher.SearcherSingleIndex
+import com.algolia.search.model.Attribute
+import com.algolia.search.model.response.ResponseSearch
+import com.algolia.search.model.search.FacetStats
+import com.algolia.search.model.search.Query
+import kotlin.reflect.KClass
+
+internal class FilterRangeConnectionSearcherImpl<T>(
+    private val filterRangeViewModel: FilterRangeViewModel<T>,
+    private val searcher: SearcherSingleIndex,
+    private val attribute: Attribute,
+    private val mapper: (Number) -> T
+) : ConnectionImpl() where T : Number, T : Comparable<T> {
+
+    val subscription: (ResponseSearch?) -> Unit = { response ->
```

Comment:
I know it's probably also not named well in other connectors elsewhere, but I would say it would be good to start giving clearer names to variables, for example `responseSubscription` so that we understand what kind of sub it is

Reply:
True, renamed for clarity, thanks :)